ui/app/AppLayouts/Chat/popups/community/SignMintTokenTransactionPopup.qml

```diff
@@ -19,6 +19,7 @@ StatusDialog {
 
     property string collectibleName
     property string networkName
+    property int chainId
```

Comment:
We don't really need this parameter in the popup context. We can just remove it!